src/en/administration/from_scratch.md

```diff
@@ -0,0 +1,886 @@
+            // API key is optional, youtube will work without it too.
+            // It is probably the same API key you use for Google Maps.
+            youtube: {
+                // api_key: "INSERT YOUR VALUE",
+                get_params: "?rel=0&showinfo=1"     // https://developers.google.com/youtube/player_parameters
+            },
+            vimeo: {
+                get_params: "?byline=0&badge=0"     // https://developer.vimeo.com/player/embedding
+            },
+
+            /*
+            soundcloud: {
+                old_player: true // enables classic player
+            },
+            giphy: {
+                media_only: true // disables branded player for gifs and returns just the image
+            }
+            */
+            /*
+            bandcamp: {
+                get_params: '/size=large/bgcol=333333/linkcol=ffffff/artwork=small/transparent=true/',
+                media: {
+                    album: {
+                        height: 472,
+                        'max-width': 700
+                    },
+                    track: {
+                        height: 120,
+                        'max-width': 700
+                    }
+                }
+            }
+            */
+        },
+
+        // WHITELIST_WILDCARD, if present, will be added to whitelist as record for top level domain: "*"
+        // with it, you can define what parsers do when they run accross unknown publisher.
+        // If absent or empty, all generic media parsers will be disabled except for known domains
+        // More about format: https://iframely.com/docs/qa-format
+
+        /*
+        WHITELIST_WILDCARD: {
+              "twitter": {
+                "player": "allow",
+                "photo": "deny"
+              },
+              "oembed": {
+                "video": "allow",
+                "photo": "allow",
+                "rich": "deny",
+                "link": "deny"
+              },
+              "og": {
+                "video": ["allow", "ssl", "responsive"]
+              },
+              "iframely": {
+                "survey": "allow",
+                "reader": "allow",
+                "player": "allow",
+                "image": "allow"
+              },
+              "html-meta": {
+                "video": ["allow", "responsive"],
+                "promo": "allow"
+              }
+        }
+        */
+
+        // Black-list any of the inappropriate domains. Iframely will return 417
+        // At minimum, keep your localhosts blacklisted to avoid SSRF
+        BLACKLIST_DOMAINS_RE: [
+            /^https?:\/\/127\.0\.0\.1/i,
+            /^https?:\/\/localhost/i,
+
+            // And this is AWS metadata service
+            // https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-metadata.html
+            /^https?:\/\/169\.254\.169\.254/
+        ]
+    };
+
+    module.exports = config;
+})();
+```
+
+### nginx script
```

Comment:
You can also leave nginx.conf out. In fact I would just link to the Lemmy repo for these files, because they will quickly get outdated here.